pkg/controller/seed-controller-manager/mla/deletion.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mla
+
+import (
+	"k8c.io/kubermatic/v2/pkg/resources"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrlruntimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func ResourcesOnDeletion(clusterNamespace string) []ctrlruntimeclient.Object {
```

Comment:
Have you tested if this cleanup is working? What I can see there are two issues:
- You are cleaning up deployment and configmap in `resources.MLANamespace`, but those objects are created in cluster namespace.
- For those three services, you've set the `name` to namespace name, and set the `namespace` to name.

Reply:
yeah... good catch, I completely screwed those names